pom.xml

```diff
@@ -1,78 +1,68 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>org.jenkins-ci.plugins</groupId>
-    <artifactId>plugin</artifactId>
-    <version>1.509.1</version><!-- which version of Jenkins is this plugin built against? -->
-  </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.3</version><!-- which version of Jenkins is this plugin built against? -->
+    </parent>
 
-  <groupId>com.puppetlabs.jenkins.plugins</groupId>
-  <artifactId>puppet-gatling-jenkins-plugin</artifactId>
-  <version>0.9.5-SNAPSHOT</version>
-  <packaging>hpi</packaging>
+    <groupId>puppetlabs</groupId>
+    <artifactId>puppet-gatling-jenkins-plugin</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+    <packaging>hpi</packaging>
 
-  <!-- get every artifact through repo.jenkins-ci.org, which proxies all the artifacts that we need -->
-  <repositories>
-    <repository>
-      <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
-    </repository>
-  </repositories>
+    <!-- get every artifact through repo.jenkins-ci.org, which proxies all the artifacts that we need -->
+    <repositories>
+        <repository>
+            <id>repo.jenkins-ci.org</id>
+            <url>http://repo.jenkins-ci.org/public/</url>
+        </repository>
+    </repositories>
 
-  <distributionManagement>
```

Comment:
Does this mean we wouldn't be publishing this plugin to nexus anymore?  Would we be publishing it somewhere else now?  Or maybe this will change to something else in a follow-up PR?